schematic_api/api/routes.py

```diff
@@ -234,7 +240,15 @@ def parse_bool(str_bool):
         raise ValueError(
             "String boolean does not appear to be true or false. Please verify input."
         )
-        
+
+def return_as_json(manifest_local_file_path):
+    manifest_csv = pd.read_csv(manifest_local_file_path)
+    try:
+        manifest_json = manifest_csv.to_dict(orient="records")
+        return manifest_json
+    except JSONDecodeError as e: 
+        raise (f"Fail to return the manifest as a json") from e
```

Comment:
@linglp You'll want to be careful here:

This raises a different error than:
```
In [2]: raise ("foo")
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [2], in <cell line: 1>()
----> 1 raise ("foo")

TypeError: exceptions must derive from BaseException
```

This:
```
In [1]: raise ValueError("foo")
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Input In [1], in <cell line: 1>()
----> 1 raise ValueError("foo")

ValueError: foo
```

Notice how the first one is an actual error with how you've used the `raise` statement.

Reply:
Thanks Tom. I have updated the code to make sure that the right error message got raised.